spec/fixtures/files/batch_import/sample-batch-import.csv

```diff
@@ -1,6 +1,6 @@
-_uid,_identifiers[0],_digital_object_type,_parents[0].uid,_parents[1].identifier,_parents[2].identifier,_primary_project.string_key,_title.value.sort_portion,_resource_imports.main.method,_resource_imports.main.location
-2f4e2917-26f5-4d8f-968c-a4015b10e50f,item1,item,,,,great_project,Existing Item 1,,
-,asset1,asset,2f4e2917-26f5-4d8f-968c-a4015b10e50f,,,great_project,New Asset 1,fixture_file,test.txt
-,asset2,asset,,item1,,great_project,New Asset 2,fixture_file,test.txt
-,asset3,asset,,item1,item2,great_project,New Asset 3,fixture_file,test.txt
-,item2,item,,,,great_project,New Item 2,,
+_assign_uid,_uid,_identifiers[0],_digital_object_type,_parents[0].uid,_parents[1].identifier,_parents[2].identifier,_primary_project.string_key,_title.value.sort_portion,_resource_imports.main.method,_resource_imports.main.location
+,2f4e2917-26f5-4d8f-968c-a4015b10e50f,item1,item,,,,great_project,Existing Item 1,,
+3f4e2917-26f5-4d8f-968c-a4015b10e50f,,asset1,asset,2f4e2917-26f5-4d8f-968c-a4015b10e50f,,,great_project,New Asset 1,fixture_file,test.txt
+4f4e2917-26f5-4d8f-968c-a4015b10e50f,,asset2,asset,,item1,,great_project,New Asset 2,fixture_file,test.txt
+5f4e2917-26f5-4d8f-968c-a4015b10e50f,,asset3,asset,,item1,item2,great_project,New Asset 3,fixture_file,test.txt
+6f4e2917-26f5-4d8f-968c-a4015b10e50f,,item2,item,,,,great_project,New Item 2,,
```

Comment:
Did this change have any testing side effects?

Reply:
i ran hyacinth:ci and all tests passed, so apparently not.

Reply:
Also, I'd recommend not changing all of the assets here to have an _assigned_uid, just to make sure we have some examples of asset creation without assigning an id.  Maybe just add _assigned_uid it to one of them and leave blank for the rest?

Reply:
After our conversation about those future csv pre-validation TODOs, I'll revise my earlier suggestion: It would still be good to have _assign_uid for only one of those rows, but it's fine not to worry about test side effects or needing to add a new test to cover the addition.  When we add csv validation improvements down the line, this will be a reminder to make sure that _assign_uid is handled appropriately during pre-validation.

Note about pre-validation: It's only meant to highlight user errors earlier in the csv processing process, but the real protections are on the existing Digital Object validations.

Reply:
I removed all but one assign_uid value since they were now unnecessary.